src/platform/PlatformTools.ts

```diff
@@ -111,40 +119,21 @@ export class PlatformTools {
                     return require("mssql");
 
                 /**
-                * other modules
-                */
-                case "mkdirp":
-                    return require("mkdirp");
-
-                case "path":
-                    return require("path");
-
-                case "debug":
-                    return require("debug");
-
-                case "app-root-path":
-                    return require("app-root-path");
-
-                case "glob":
-                    return require("glob");
-
-                case "typeorm-aurora-data-api-driver":
-                    return require("typeorm-aurora-data-api-driver");
-                /**
-                * default
-                */
-                default:
-                    return require(name);
-
+                 * react-native-sqlite
+                 */
+                case "react-native-sqlite-storage":
+                    return require("react-native-sqlite-storage");
             }
 
         } catch (err) {
-            if (!path.isAbsolute(name) && name.substr(0, 2) !== "./" && name.substr(0, 3) !== "../") {
```

Comment:
not sure why we had this check before (I think it was a PR), are you sure we don't need it anymore?

Reply:
This check was for the following case:

1. We are running the global typeorm
2. We want a module installed in a local project
3. We want to check to make sure we aren't trying to load a relative file

Because `PlatformTools.load` is now limited in what it can require through a specific array of modules we cannot have relative files being loaded.

As such, leaving this check in would have been fine but it would never be true - the module name cannot have these values.